index.html

```diff
@@ -29,23 +33,193 @@ <h1>Hey There. <br> I'm Bless.</h1>
                 <h2>I'm a Software Developer</h2>
                 <p class="text">
                     I can help you build a product , feature
-                    or website Look through some of my 
-                    work and experience! If you like what you 
-                    see and have a project you need coded, 
+                    or website Look through some of my
+                    work and experience! If you like what you
+                    see and have a project you need coded,
                     don’t hesitate to contact me.
                 </p>
                 <div class="icon">
                     <ul>
-                        <li><a href="https://github.com/BlessmanNewton"><i class="fa fa-github" aria-hidden="true"></i></a></li>
-                        <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-                        <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
-                        <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter" aria-hidden="true"></i></i></a></li>
-                        <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+                        <li><a href="https://github.com/BlessmanNewton"><i class="fa fa-github"
+                                    aria-hidden="true"></i></a></li>
+                        <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin"
+                                    aria-hidden="true"></i></a></li>
+                        <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist"
+                                    aria-hidden="true"></i></a></li>
+                        <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter"
+                                    aria-hidden="true"></i></i></a></li>
+                        <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium"
+                                    aria-hidden="true"></i></a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        </div>
+    </section>
+    <section>
+        <div class="work-section">
+            <div class="recent-work">
+                <h3>My Recent Work</h3>
+                <hr>
+            </div>
+            <div class="project-box">
+                <img src="images/Img Placeholder.png" alt="">
```

Comment:
- [ ] Kindly ensure to provide the `alt text`. Alt text is important because it makes images more accessible to people and search engines. Even when they cannot be displayed ☺️.